src/app/space/create/apps/apps.component.html

```diff
@@ -0,0 +1,99 @@
+<div id="apps" class="">
+
+  <div class="cards-pf">
+    <div class="row row-cards-pf">
+      <div class="col-xs-12 col-sm-4 col-md-2"><h3><strong>Applications</strong></h3></div>
+      <div class="col-xs-12 col-sm-4 col-md-2"><h3><strong>Environments</strong></h3></div>
+    </div>
+  </div>
+
+  <div class="cards-pf">
+    <div class="row row-cards-pf">
+      <div class="col-xs-12 col-sm-4 col-md-2"></div>
+      <div class="col-xs-12 col-sm-4 col-md-3" *ngFor="let environment of environments | async">
+        <h4 style="text-transform: capitalize;">{{ environment.name }}</h4>
+      </div>
+    </div>
+  </div>
+
+  <div class="cards-pf" *ngFor="let application of applications | async">
+    <div class="container-fluid container-cards-pf">
+      <div class="row row-cards-pf">
+        <div class="col-xs-6 col-sm-4 col-md-2">
+          <h4>{{ application }}</h4>
+        </div>
+        <app-card class="col-xs-6 col-sm-4 col-md-3" [applicationId]="application" [environment]="environment"
+          *ngFor="let environment of environments | async"></app-card>
+      </div>
+    </div>
+  </div>
+
+  <div class="row panel panel-default">
+    <div class="panel-body cards-pf" style="background: lightgrey; padding-top: 0; padding-bottom: 0; margin-top: 0; margin-bottom: 0;">
+      <div class="container-fluid container-cards-pf" style="padding-top: 0; padding-bottom: 0; margin-top: 0; margin-bottom: 0;">
+        <div class="row row-cards-pf">
+          <div class="col-xs-12 col-sm-4 col-md-2">
+            <h3><strong><a style="color: black; cursor: pointer;"
+              (click)="resourcesCollapsed = !resourcesCollapsed"><span class="fa fa-angle-down"></span>&nbsp;&nbsp;Resource Usage</a></strong></h3>
+          </div>
+          <div [collapse]="resourcesCollapsed">
+            <div class="col-xs-12 col-sm-4 col-md-3" *ngFor="let environment of environments | async">
+              <div class="card-pf">
+                <div class="card-pf-body" style="font-size: 12px;">
```

Comment:
Can you add a class, instead of an inline style? Also, instead of `px`, `em` is preferred.